Platform/QemuBoardPkg/BoardConfig.py

```diff
@@ -23,16 +23,19 @@ def __init__(self, *args, **kwargs):
 
         super(Board, self).__init__(*args, **kwargs)
 
-        EXECUTE_IN_PLACE          = 0
+        # 0: QEMU  1: Simics support ->QSP
+        self.ENABLE_SIMICS        = 0
+
+        EXECUTE_IN_PLACE          = 0 if self.ENABLE_SIMICS != 1 else 1
         FREE_TEMP_RAM_TOP         = 0x70000
 
-        self.VERINFO_IMAGE_ID          = 'SB_QEMU '
+        self.VERINFO_IMAGE_ID          = 'SB_QEMU ' if self.ENABLE_SIMICS != 1 else 'SBL_QSP '

```

Comment:
Maybe keep them same ?   Just treat Simics as a special board for QEMU platform ?

Reply:
Done.